<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 68px;
line-height: 82px;
display: flex;
align-items: center;
text-align: center;
text-decoration-line: underline;
background: #1E1F2E;
color: #FFF;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:68px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:68px; text-decoration: underline;&quot;&gt;ADMIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 68px;
line-height: 82px;
color: #252748;
border: none;
background: #f0f0f0;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:68px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:36pt; font-weight:700; text-decoration: underline;&quot;&gt;Latest Purchases&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 900;
font-size: 36px;
line-height: 44px;

color: #FFFFFF;border:none;
background: #BB5858</string>
   </property>
   <property name="text">
    <string>Manage Admin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>0</y>
     <width>876</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 876px;
height: 199px;
left: 404px;
top: 0px;

background: #252748;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 900;
font-size: 36px;
line-height: 44px;
border:none;

color: #FFFFFF;
background: #252748;</string>
   </property>
   <property name="text">
    <string>ADMIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
position: absolute;
width: 404px;
height: 720px;
left: 0px;
top: 0px;

background: #1E1F2E;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>590</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 500;
font-size: 42px;
line-height: 82px;
color: #252748;
border:none;
background: #f0f0f0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>876</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #f0f0f0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 500;
font-size: 42px;
line-height: 82px;
color: #252748;border:none;

background: #f0f0f0;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>260</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 900;
font-size: 36px;
line-height: 44px;

color: #FFFFFF;
background: #252748;</string>
   </property>
   <property name="text">
    <string>Reports</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 900;
font-size: 36px;
line-height: 44px;
border:none;

color: #FFFFFF;
background: #BB5858</string>
   </property>
   <property name="text">
    <string>Manage Player</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Inter';
font-style: normal;
font-weight: 500;
font-size: 42px;
line-height: 82px;
color: #252748;
border:none;
background: #f0f0f0;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>30</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 900;
font-size: 36px;
line-height: 44px;
border:none;
color: #FFFFFF;
background: #BB5858</string>
   </property>
   <property name="text">
    <string>Refund</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 200;
font-size: 25px;
line-height: 44px;
border:none;overflow-wrap: break-word;

color: #FFFFFF;
background: #BB5858</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 200;
font-size: 25px;
line-height: 44px;
border:none;overflow-wrap: break-word;

color: #FFFFFF;
background: #BB5858</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: 'Inter';
font-weight: 200;
font-size: 25px;
line-height: 44px;
border:none;overflow-wrap: break-word;

color: #FFFFFF;
background: #BB5858</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>textEdit</zorder>
  <zorder>textEdit_5</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>lineEdit_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
